breathing_cat/resources/sphinx/conf.py.in

```diff
@@ -142,92 +132,13 @@ html_css_files = [
 # html_sidebars = {}
 
 
-# -- Options for HTMLHelp output ---------------------------------------------
-
-# Output file base name for HTML help builder.
-htmlhelp_basename = "@PROJECT_NAME@_doc"
-
-
-# -- Options for LaTeX output ------------------------------------------------
-
-latex_elements = {  # type: ignore
-    # The paper size ('letterpaper' or 'a4paper').
-    #
-    # 'papersize': 'letterpaper',
-    # The font size ('10pt', '11pt' or '12pt').
-    #
-    # 'pointsize': '10pt',
-    # Additional stuff for the LaTeX preamble.
-    #
-    # 'preamble': '',
-    # Latex figure (float) alignment
-    #
-    # 'figure_align': 'htbp',
-}
-
-# Grouping the document tree into LaTeX files. List of tuples
-# (source start file, target name, title,
-#  author, documentclass [howto, manual, or own class]).
-latex_documents = [
-    (
-        master_doc.encode("utf-8"),
-        "@PROJECT_NAME@.tex",
-        "@PROJECT_NAME@ Documentation",
-        author,
-        "manual",
-    ),
-]
-
-
-# -- Options for manual page output ------------------------------------------
-
-# One entry per manual page. List of tuples
-# (source start file, name, description, authors, manual section).
-man_pages = [
-    (master_doc, "@PROJECT_NAME@", "@PROJECT_NAME@ Documentation", [author], 1)
-]
-
-
-# -- Options for Texinfo output ----------------------------------------------
-
-# Grouping the document tree into Texinfo files. List of tuples
-# (source start file, target name, title, author,
-#  dir menu entry, description, category)
-texinfo_documents = [
-    (
-        master_doc,
-        "@PROJECT_NAME@",
-        "@PROJECT_NAME@ C++",
-        author,
-        "@PROJECT_NAME@",
-        "One line description of project.",
-        "Robotics",
-    ),
-]
-
-
-# -- Options for Epub output -------------------------------------------------
-
-# Bibliographic Dublin Core info.
-epub_title = project
-
-# The unique identifier of the text. This can be a ISBN number
-# or the project homepage.
-#
-# epub_identifier = ''
-
-# A unique identification for the text.
-#
-# epub_uid = ''
-
-# A list of files that should not be packed into the epub file.
-epub_exclude_files = ["search.html"]
-
-
 # -- Extension configuration -------------------------------------------------
 
 # -- Options for intersphinx extension ---------------------------------------
 
+# TODO intersphinx doesn't make sense like this.  Either packages need some way to add
```

Comment:
maybe a config file in the doc folder?
`intersphinx.txt`?

Reply:
Yes, either something like this simply some Python file that gets appended to conf.py (would be more flexible and also allow adding/changing other settings).  See #10.